WordPress/Classes/ViewRelated/WhatsNew/Data store/AnnouncementsStore.swift

```diff
@@ -124,14 +124,12 @@ class CachedAnnouncementsStore: AnnouncementsStore {
 private extension CachedAnnouncementsStore {
 
     func cacheIsValid(for announcements: [Announcement]) -> Bool {
-        guard let minimumVersion = announcements.first?.minimumAppVersion,   // there should not be more than one announcement
-              let maximumVersion = announcements.first?.maximumAppVersion,   // per version, but if there is, each of them must match the version
-              let targetVersions = announcements.first?.appVersionTargets,   // so we might as well choose the first
-              let version = versionProvider.version,
-              ((minimumVersion...maximumVersion).contains(version) || targetVersions.contains(version)) else {
+        guard let announcement = announcements.first, let version = versionProvider.version else {
             return false
         }
-        return true
+
+        return (announcement.minimumAppVersion...announcement.maximumAppVersion).contains(version)
+        || announcement.appVersionTargets.contains(version)
```

Comment:
TODO, keep the comment because it explains why we look at the first only (we could filter, though, no?)